app/services/users.js

```diff
@@ -0,0 +1,10 @@
+const { databaseError, defaultError } = require('../errors');
+const userModel = require('../models').User;
+
+exports.signUp = user =>
+  userModel.create(user).catch(error => {
+    if (error.name === 'SequelizeUniqueConstraintError') {
+      throw defaultError('Email is already in use');
```

Comment:
This is not a default error. And you create a `signUpError` in errors file but is not used in the api